Formula/e/envelope.rb

```diff
@@ -0,0 +1,35 @@
+class Envelope < Formula
+  desc "Environment variables CLI tool"
+  homepage "https://github.com/mattrighetti/envelope"
+  url "https://github.com/mattrighetti/envelope/archive/refs/tags/0.3.11.tar.gz"
+  sha256 "1a378564b07e041fbf3212655e8c6442f8973080cf0698886764ce38982661cc"
+  license :public_domain
```

Comment:
I did not find your public domain license declaration in the repo, can you add it? Thanks!

Reply:
Made it `Unlicense` as I've seen CC is not really intended for software

Reply:
yeah, exactly